setup.py

```diff
@@ -64,7 +64,7 @@ def read(*parts):
     install_requires=install_requirements,
     extras_require={
         'tests': test_requirements,
-        'filer': ['django-filer', ],
+        'filer': ['django-filer==1.2.4', ],
```

Comment:
Pinning to a specific version is the safest option, though a range (keeping it in the 1.2.x) could we're reasonably sure django-filer won't introduce breaking changes inside of a minor (i.e., 1.2.x) series.

As it is now (unpinned), when you `pip install django-markymark`, you get the latest django-filer which is incompatible with the current version of markymark. Pinning to 1.1 (or the 1.1.x range) would prevent the mismatch.